src/Microsoft.ML.Api/MapTransform.cs

```diff
@@ -4,6 +4,7 @@
 
 using System;
 using System.IO;
+using System.Linq;
```

Comment:
Was this leftover from something? I don't see that LINQ gets used anywhere here.